project/src/components/place-card-favorites-list/place-card-list-by-cities.tsx

```diff
@@ -22,6 +23,7 @@ export default function PlaceCardListByCities({favoritesByCities}: PlaceCardList
           </div>
           <PlaceCardList
             offers={favoritesByCities[city]}
+            onCurrentOfferChange = {onCurrentOfferChange}
```

Comment:
Пропсы форматируем только так 
```suggestion
            onCurrentOfferChange={onCurrentOfferChange}
```

Reply:
done